app/async_consumer.py

```diff
@@ -52,12 +52,12 @@ def connect(self):
             self._url = settings.RABBIT_URLS[server_choice]
 
             try:
-                logger.info('Connecting', amqp_url=self._url, attempt=count)
+                logger.info('Connecting to rabbit', attempt=count)
```

Comment:
should this say queue to be more in line with the messages output in the other file?